lib/carbon/conf.py

```diff
@@ -433,11 +434,11 @@ def postOptions(self):
             self["aggregation-rules"] = join(settings["CONF_DIR"], settings['AGGREGATION_RULES'])
         settings["aggregation-rules"] = self["aggregation-rules"]
 
-        if settings["RELAY_METHOD"] not in ("rules", "consistent-hashing", "aggregated-consistent-hashing"):
-            print ("In carbon.conf, RELAY_METHOD must be either 'rules' or "
-                   "'consistent-hashing' or 'aggregated-consistent-hashing'. Invalid value: '%s'" %
-                   settings.RELAY_METHOD)
-            sys.exit(1)
+        router = settings["RELAY_METHOD"]
+        if router not in DatapointRouter.plugins:
+            print "No router plugin implemented for '%s'" % router
+            print "Available plugins: '%s'" % ', '.join(DatapointRouter.plugins)
```

Comment:
gFixed. New message looks like `In carbon.conf, RELAY_METHOD must be one of rules, consistent-hashing, fast-aggregated-hashing, fast-hashing, aggregated-consistent-hashing. Invalid value: 'consistent-hashing'`